bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZpTo3mu/PROCESSNAME_run_card.dat

```diff
@@ -0,0 +1,274 @@
+     6500     = ebeam1  ! beam 1 total energy in GeV
+     6500     = ebeam2  ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ 'lhapdf'    = pdlabel     ! PDF set                                  
+ 263400    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number 
+
+#*********************************************************************
+# Renormalization and factorization scales                           *
+#*********************************************************************
+ F        = fixed_ren_scale  ! if .true. use fixed ren scale
+ F        = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.1880  = scale            ! fixed ren scale
+ 91.1880  = dsqrt_q2fact1    ! fixed fact scale for pdf1
+ 91.1880  = dsqrt_q2fact2    ! fixed fact scale for pdf2
+ 1        = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+  0	= ickkw ! 0 no matching, 1 MLM, 2 CKKW matching
+  1	= highestmult ! for ickkw=2, highest mult group
+  1	= ktscheme ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+  1.0	= alpsfact ! scale factor for QCD emission vx
+  F	= chcluster ! cluster only according to channel diag
+  T	= pdfwgt ! for ickkw=1, perform pdf reweighting
+ 263401   = PDF_set_min      ! First of the error PDF sets 
+ 263500   = PDF_set_max      ! Last of the error PDF sets
+  4	= asrwgtflavor ! highest quark flavor for a_s reweight
+  T	= clusinfo ! include clustering tag in output
+  3.0	= lhe_version ! Change the way clustering information pass to shower.        
+#*********************************************************************       
+#*********************************************************************
+#**********************************************************
+#
+#**********************************************************
+# Automatic ptj and mjj cuts if xqcut > 0
+# (turn off for VBF and single top processes)
+#**********************************************************
+   T  = auto_ptj_mjj  ! Automatic setting of ptj and mjj
+#**********************************************************
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15000  = bwcutoff      ! (M+/-bwcutoff*Gamma)
```

Comment:
there is always discussion on the bwcutoff, from the MG convention, when the decay chain is used (just as your process), the bwcutoff will be used to define the on-shell resonance, in which a narrow width limit is required. Very large bwcutoff will spoil the narrow width limit and should be avoided.